docs/markdown/Commands.md

```diff
@@ -0,0 +1,156 @@
+
+`TARGET` has the following syntax: `[PATH/]NAME[:TYPE]`.
+`NAME`: name of the target from `meson.build` (e.g. `foo` from `executable('foo', ...)`).
+`PATH`: path to the target relative to the root `meson.build` file. Note: relative path for a target specified in the root `meson.build` is `./`.
+`TYPE`: type of the target (e.g. `shared_library`, `executable` and etc)
+
+`PATH` and/or `TYPE` can be ommited if the resulting `TARGET` can be used to uniquely identify the target in `meson.build`.
+
+For example targets from the following code:
+```meson
+shared_library('foo', ...)
+static_library('foo', ...)
+executable('bar', ...)
+```
+can be invoked with `meson compile foo:shared_library foo:static_library bar`.
+
+Note: the following limitations for backends apply:
+- `ninja`: `custom_target` is not supported.
+- `vs`: `run_target` is not supported.
+
+Notable options:
+- `--jobs JOBS`: The number of worker jobs to run (if supported). If the value is less
+                 than 1 the build program will guess.
+- `--clean`: Cleans the build directory.
+
+### dist
+
+*(since 0.52.0)*
+
+Generates a release archive from the current source tree.
+
+Example: `meson dist -C builddir`
+
+This creates a file called `projectname-version.tar.xz` in the build
+tree subdirectory `meson-dist`. This archive contains the full
+contents of the latest commit in revision control including all the
+submodules (recursively). All revision control metadata is removed.
+Meson then takes
+this archive and tests that it works by doing a full compile + test +
+install cycle. If all these pass, Meson will then create a SHA-256
+checksum file next to the archive.
+
+**Note**: Meson behaviour is different from Autotools. The Autotools
+"dist" target packages up the current source tree. Meson packages
+the latest revision control commit. The reason for this is that it
+prevents developers from doing accidental releases where the
+distributed archive does not match any commit in revision control
+(especially the one tagged for the release).
+
+Noteable options:
+- `--include-subprojects`: Include source code of subprojects that have been used for the build.
+
+### init
+
+*(since 0.45.0)*
+
+Generates a basic meson setup from the template.
+
+Example: `meson init -C sourcedir`
+
+Notable options:
+- `--name NAME`: project name. default: name of current directory.
+- `--deps DEPS`: list of project dependencies, comma-separated.
+
+### introspect
+
+Displays information about a configured meson project.
+
+Syntax: `meson introspect [builddir] INTROSPECT_COMMAND [introspect_command_options]`  
+Example: `meson introspect builddir`
+
+Notable introspect commands:
+- `--buildoptions`: List all build options.
+- `--dependencies`: List external dependencies.
+- `--targets`: List top level targets.
+
+### install
+
+*(since 0.47.0)*
+
+Installs the project to the folder specified in `setup`.
+
+Example: `meson install -C builddir`
+
+See [this page](Installing.md) for more info.
```

Comment:
Links using text like `this page` is considered an anti-pattern because it makes it harder for blind users to see what the link is for. Should use something like `See [the installation documentation](Installing.md) for more info`. Same for below.

Reply:
Fixed